.vimrc

```diff
@@ -436,3 +452,33 @@ let g:matchup_matchparen_deferred = 1
 " Diminactive
 " ============================================================================
 let g:diminactive_enable_focus = 1
+
+" ============================================================================
+" ANYFOLDS
+" ============================================================================
+autocmd Filetype * AnyFoldActivate   
+set foldlevel=99 " Open all folds
+
+" ============================================================================
+" LENGTHMATTERS
+" ============================================================================
+let g:lengthmatters_start_at_column = 88
```

Comment:
This should be in a local file as this depends per project/user.

Reply:
I am not sure, since I would like to have it as default length for the team